src/components/Page/components/Header/components/Title/Title.scss

```diff
@@ -3,47 +3,35 @@
 .Title {
   @include text-breakword;
   @include text-emphasis-strong;
-  font-size: rem(24px);
-  line-height: rem(28px);
+  font-size: 24px;
+  line-height: 28px;
 
   @include when-typography-not-condensed {
-    font-size: rem(20px);
+    font-size: 20px;
```

Comment:
@alex-page  Random late question since I noticed this while looking into some typography stuff. Is there a reason this isn't using the typography tokens?

Feel free to ignore me if so, but it seemed odd since everything else I looked at seems to be.

Reply:
We are going to back through the values and update them to use tokens now. Thanks for noticing this one!